docs/MaintainerVersioning.md

```diff
@@ -269,19 +165,19 @@ You can specify pre-release versions for plug-ins using one of the following met
     **Example:**
 
     ```
-    @brightside/imperative": "1.1.0-next.201808241438"
+    @zowe/imperative": "8.0.0-next-202408131445"
     ```
 
     Such a version matches only this one snapshot of the Zowe CLI or Imperative pre-release. The plug-in will have to change the version string with each new pre-release snapshot. This requires frequent updates to the plug-in's package.json file, but helps to ensure that the plug-in will never be accidentally compatible with a later stable version of that dependency.
 
-* The plug-in can specify that it wants any version of the peerDependency starting with the earliest version of the current pre-release and any later version going forward.
+* The plug-in can specify that it wants any version of the peerDependency starting with the earliest version of the current pre-release and any later version going forward. This includes all GA versions of the peerDependency with the same major version, starting at 8.0.0:
 
     **Example:**
 
     ```
-    @brightside/imperative": ">=1.1.0-0
+    @zowe/imperative": ">=8.0.0-next-202408131445 <9.0.0"
```

Comment:
I think that the content is good. I think you will want to an opening quote (") before
```
    @zowe/imperative": ...
```
in both this example and the previous example.